frontend/src/components/compound/Banner/SpaceBanner.tsx

```diff
@@ -1,18 +1,19 @@
 const SpaceBanner = () => {
-    return <div className="flex flex-col p-4 border border-2 rounded-lg bg-[#F7FCEF] gap-2 m-6">
-        <div className="text-md lg:text-base font-bold">
-
-            🏡 Have a co-living space, sublet, or vacant room?
```

Comment:
you're deleting a lot of whitespace in these changes - i wonder if you have a different formatter than we do in VSCode, like Prettier. tbh i feel like it's fine